server/issue.go

```diff
@@ -728,6 +728,20 @@ func (p *Plugin) getIssueDataForCloudWebhook(instance Instance, issueKey string)
 	return issue, nil
 }
 
+func (p *Plugin) getEpicName(epicID string, instanceID types.ID) (string, error) {
+	instance, err := p.instanceStore.LoadInstance(instanceID)
+	if err != nil {
+		return "", err
+	}
+
+	epic, err := p.getIssueDataForCloudWebhook(instance, epicID)
+	if err != nil {
+		return "", err
+	}
+
+	return epic.Fields.Summary, err
```

Comment:
Just for readability

```suggestion
	return epic.Fields.Summary, nil
```

Reply:
I think we actually want the `Epic Name` field, not the `Summary`. They are two different fields, and the `Epic Name` is actually what users see in Jira when applying an epic link to an issue. The `Summary` of an epic is potentially a longer name that is sometimes neglected and not as useful as the `Epic Name`.

The `Epic Name` field is likely a custom field. If it's not trivial to get that field by name, then we can use the summary.